src/windowing/meson.build

```diff
@@ -9,24 +9,35 @@ libwindowing_deps = [
   dep_xfce4windowing,
 ]
 
-libwindowing = static_library(
-  'windowing',
+libwindowing = shared_library(
+  'budgie-windowing',
   libwindowing_sources,
   dependencies: libwindowing_deps,
   vala_args: [
     '--pkg', 'gio-unix-2.0',
     '--pkg', 'gtk+-3.0',
     '--pkg', 'libxfce4windowing-0',
-    '--vapidir', join_paths(meson.source_root(), 'vapi'),
+    '--vapidir', join_paths(meson.project_source_root(), 'vapi'),
   ],
+  version: '0.0.0',
+  install: true,
+)
+
+# Allow building against libbudgie-windowing
+pkgconfig.generate(
+    name: 'Budgie Windowing',
+    description: 'Budgie Windowing Library',
+    version: '1',
+    filebase: 'budgie-windowing-1.0',
+    libraries: ['-L${libdir}', '-lbudgie-windowing'],
+    requires: ['gtk+-3.0 >= 3.24.0']
```

Comment:
Should it also require libxfce4windowing?

Reply:
That's line 9 isn't it?

Reply:
Does that automatically get put in the .pc?

I guess, does this even matter? xD